src/Language/PureScript/AST/Declarations.hs

```diff
@@ -260,6 +260,33 @@ instance Eq DeclarationRef where
   r == (PositionedDeclarationRef _ _ r') = r == r'
   _ == _ = False
 
+-- enable sorting lists of explicitly imported refs
+instance Ord DeclarationRef where
+  compare = compDecRef
+
+compDecRef :: DeclarationRef -> DeclarationRef -> Ordering
+compDecRef (TypeRef name _) (TypeRef name' _) = compare name name'
+compDecRef (TypeOpRef name) (TypeOpRef name') = compare name name'
+compDecRef (ValueRef ident) (ValueRef ident') = compare ident ident'
+compDecRef (ValueOpRef name) (ValueOpRef name') = compare name name'
+compDecRef (TypeClassRef name) (TypeClassRef name') = compare name name'
+compDecRef (TypeInstanceRef ident) (TypeInstanceRef ident') = compare ident ident'
+compDecRef (ModuleRef name) (ModuleRef name') = compare name name'
+compDecRef (ReExportRef name _) (ReExportRef name' _) = compare name name'
+compDecRef (PositionedDeclarationRef _ _ ref) (PositionedDeclarationRef _ _ ref') = compare ref ref'
```

Comment:
I think we just want to drop the position info for the compare instance, so I'd prefer two cases here:

```haskell
compDecRef (PositionedDeclarationRef _ _ ref) ref' = compare ref ref'
compDecRef ref (PositionedDeclarationRef _ _ ref') = compare ref ref'
```

This way we get the expected result when only one of the two declarations carries position info.